update_scripts.sh

```diff
@@ -4,27 +4,23 @@ echo
 echo "Fetch the latest scripts from GitHub - Navid200"
 echo
 
+. /etc/git-parameters.sh
+
 cd /tmp
 
-if [ -s ./nightscout-vps ] # Main
-#if [ -s ./cgm-remote-monitor ] # Navid's
+if [ -s ./"$GIT_REPOSITRY" ] # Main
 then
-sudo rm -r nightscout-vps # If the directory already exists in the tmp directory, delete it.  Main
-#sudo rm -r cgm-remote-monitor # If the directory already exists in the tmp directory, delete it.  Navid's
+sudo rm -r "$GIT_REPOSITRY" # If the directory already exists in the tmp directory, delete it.  Main
 fi
 
-sudo git clone https://github.com/jamorham/nightscout-vps.git # Clone the install repository.  Main
-#sudo git clone https://github.com/Navid200/cgm-remote-monitor.git # Clone the install repository.  Navid's
-cd nightscout-vps # Main
-#cd cgm-remote-monitor # Navid's
-sudo git checkout vps-1 # Main
-#sudo git checkout VerifyVM_Test # Navid's
+sudo git clone https://github.com/"$GIT_USER"/"$GIT_REPOSITRY".git # Clone the install repository.  Main
```

Comment:
We should never get the scripts from the forked repositories.  The scripts should only come from one repository, the one that we have 100% control over, Jon's.

Reply:
If a user creates a fork and in it, they modify the ConfigureFreedns.sh script, and then, they run the update_scripts command, none of the changes they made to the ConfigureFreedns.sh will come through.

The only reason we allow people to install from a fork is so that they can customize Nightscout, not our scripts.